DOCUMENTATION.md

```diff
@@ -191,15 +191,21 @@ For example:
 email_reply_to_id='8e222534-7f05-4972-86e3-17c5d9f894e2' # optional UUID string
 ```
 
-You can leave out this argument if your service only has one email reply to address, or you want to use the default email address.
+You can leave out this argument if your service only has one email reply-to address, or you want to use the default email address.
 
-### Upload a document
+### Send a document by email
 
-You can upload a document to link to from an email notification.
+To send a document by email, upload a file then add a placeholder field to the template.
 
-The document must be a PDF file smaller than 2MB.
+The placeholder field contains a secure link to download the document instead of an attachment.
```

Comment:
"will contain" or "will be replaced with a link"? I feel like this sentence is trying to say more than one thing too: what happens with the placeholder value and that it's different from attachments.

Reply:
I agree that the sentence is trying to do too many things.

User research showed that we need to explain that this meets the need for email attachments, but is not _actually_ email attachments.

Once we have some feature content for _send a document_ on Notify, I'd like to include a link to that here. Eg. "Find out why we use links instead of attachments." However, until that's ready we need an alternative solution.

An earlier iteration of the _send a document_ section opened with the line `Send files without the need for email attachments. 
To send a document by email...` 

We could revert to that and then change this line to `The placeholder field is replaced by a secure link to download the document.`

What do you think @allait @jonathanglassman?